### DISCUSSION TITLE
Welcome to GitProfile Discussions!

### DISCUSSION THREAD

**Discussion by arifszn** [arifszn/gitprofile#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by gyanish0** [comment]
i get this warning how to fix

WARNING in ./node_modules/@arifszn/gitprofile/dist/gitprofile.es.js 3079:6-41
Module not found: Error: Can't resolve './locale' in '/home/lz-12/git-user-clone/node_modules/@arifszn/gitprofile/dist'

**Reply by arifszn** [reply]
> i get this warning how to fix
> 
> WARNING in ./node_modules/@arifszn/gitprofile/dist/gitprofile.es.js 3079:6-41 Module not found: Error: Can't resolve './locale' in '/home/lz-12/git-user-clone/node_modules/@arifszn/gitprofile/dist'

I guess you are using it as a package on next.js. This is just a warning of `moment` on next.js and should not cause any issue from using it.

**Reply by gyanish0** [reply]
No I am not using next.js I am using simple react js

**Reply by gyanish0** [reply]
I am not able to deploy on netlify due to this warning please help me how to fix

**Reply by arifszn** [reply]
Moving the conversation [here](https://github.com/arifszn/gitprofile/issues/76).

**Comment by zakery1369** [comment]
Is it possible to add more social networks?
Like Skype and Telegram ??

**Reply by arifszn** [reply]
@csarnataro can you please look at it if you have time?

**Reply by csarnataro** [reply]
Sure, I can have a look in the next few days.

**Reply by arifszn** [reply]
@zakery1369 added in [v2.4.0](https://github.com/arifszn/gitprofile/releases/tag/v2.4.0).

**Comment by notslimboy** [comment]
is it possible to give option hide a github project? since i'am a game designer that barely working on github project?

**Reply by arifszn** [reply]
Yes, I am working on this. Expect an update within a week.

**Reply by arifszn** [reply]
The feature has been added. https://github.com/arifszn/gitprofile?tab=readme-ov-file#projects

Set the `display` property to false.

**Reply by notslimboy** [reply]
Thanks a lot for this new feature

**Comment by lindaboshans** [comment]
Would it be possible to include a feature/section for linking publications that are on pubmed?

**Reply by arifszn** [reply]
I do not have enough knowledge about PubMed. Any PR is welcome though.

**Reply by arifszn** [reply]
@lindaboshans, how about manually giving entries of the publication in `gitprofile.config`? Can you please give some inputs on https://github.com/arifszn/gitprofile/discussions/515?

**Comment by lindaboshans** [comment]
Would it also be possible to make the images for external projects a little larger? they are quite small and I think it would look better visually.

**Reply by arifszn** [reply]
Done.
